### PR TITLE
Allow `handlerInstallPath` to work without `stateStore`

### PR DESCRIPTION
**Pull request by rockingskier**
###  Summary
#1506

This is a bit of a guess.  It worked when I edited the compiled code in my `node_modules` 🤮 but I wouldn't be surprised if there was a better way :)

I can't run the tests locally so I'm fully expecting them to fail.  This is more of a "here's what I found" PR.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/oauth/src/install-provider.ts [hunk 1]
```diff
@@ -350,30 +346,33 @@ export class InstallProvider {
...
-      if (res.getHeader('Set-Cookie')) {
-        // If the cookies already exist
-        const existingCookies = res.getHeader('Set-Cookie') || [];
-        const allCookies: string[] = [];
-        if (Array.isArray(existingCookies)) {
-          allCookies.push(...existingCookies);
-        } else if (typeof existingCookies === 'string') {
-          allCookies.push(existingCookies);
+      let state;
+      if (this.stateStore) {
```
Comment (seratch) [inline packages/oauth/src/install-provider.ts:350]: Can you use `this.stateVerification` flag instead and place L340 - L342 that you've deleted inside the if statement?
```suggestion
      if (this.stateVerification) {
```

File: packages/oauth/src/install-provider.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ import defaultRenderHtmlForInstallPath from './default-render-html-for-install-p
 import { InstallPathOptions } from './install-path-options';
 
 /**
- * InstallProvider Class. Refer to InsallProviderOptions interface for the details of constructor arguments.
+ * InstallProvider Class. Refer to InstallProviderOptions interface for the details of constructor arguments.
```
Comment (seratch) [inline packages/oauth/src/install-provider.ts:30]: Thanks for detecting errors in the comments!

**Review by seratch — CHANGES_REQUESTED** [review]
Could you check my comments?

File: packages/oauth/src/install-provider.ts (hunk 2 above)
Comment (rockingskier) [reply packages/oauth/src/install-provider.ts:30]: No worries, my IDE does most of the work :)

File: packages/oauth/src/install-provider.ts (hunk 1 above)
Comment (rockingskier) [reply packages/oauth/src/install-provider.ts:350]: I did that initially but it makes TS sad due to `this.stateStore` being optional.
Screenshot 2022-07-04 at 07 30 31

I can do: 
* `if (this.stateStore)`
* `if(this.stateVerification && this.stateStore)
* `await this.stateStore?.generateStateParam` but this gets complicated as `state` then becomes optional

Comment (rockingskier) [reply packages/oauth/src/install-provider.ts]: Sounds good 👍  Done.

**Review by seratch — CHANGES_REQUESTED** [review]
Can you resolve the TS compilation error?

**Comment by rockingskier** [comment]
Sorted my local env and now I'm not coding blind :)

Fixed, tests passing locally. Hopefully all good to go.

**Review by seratch — APPROVED** [review]
Looks great to me! Thanks for contributing to this SDK!

**Pull request merged**